gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -433,6 +433,10 @@
             <property name="format" value="^[a-z][a-zA-Z0-9]+$"/>
             <message key="name.invalidPattern" value="Parameter name ''{0}'' must match pattern ''{1}''."/>
         </module>
+        <module name="LambdaParameterName"> <!-- Java Style Guide: Parameter names -->
```

Comment:
Adds parameter name checkstyle for lambda parameters. There's issues with ParameterName right now for lambdas that it doesn't correctly detect lambdas (and after applying this I can confirm that I have few violations of lambda parameter names that weren't caught before). From 8.12 checkstyle ParameterName will completely ignore lambda parameters and this check is preferred.